code/modules/mob/living/carbon/human/species/shadekin/shadekin_trait.dm

```diff
@@ -0,0 +1,89 @@
+/datum/trait/kintype
+	allowed_species = list(SPECIES_SHADEKIN)
+	var/color = BLUE_EYES
+	name = "Shadekin Blue Adaptation"
+	desc = "Makes your shadekin adapted as a Blue eyed kin! This gives you decreased energy regeneration in darkness, decreased regeneration in the light amd unchanged health!"
+	cost = 0
+	var_changes = list(	"total_health" = 100,
+						"energy_light" = 0.5,
+						"energy_dark" = 0.5,
+						"unarmed_types" = list(/datum/unarmed_attack/stomp, /datum/unarmed_attack/kick,/datum/unarmed_attack/shadekinharmbap))
```

Comment:
Using any of the traits removes bite/claw attack options, replacing them entirely with harmbap. Is that intentional? I know shadekin had special variations of those for energy gain, but it makes sense to have regular left in at least as an option for fluff. Although, admittedly, I don't get the purpose of replacing either with new separate attack in the first place, especially if said attack doesn't contribute to replication of oldschool "different kins gain different levels of energy via attacks" thing that simplemob kin had.

Reply:
The custom attack for kin is flavored as a claw attack but without causing bleed.